grace/io/image_dataset.py

```diff
@@ -30,6 +34,12 @@ class ImageGraphDataset(Dataset):
         Transformation added to the images and targets
     image_filetype : str
         File extension of the image files
+    keep_unknown_labels : bool
```

Comment:
is this configured somewhere or only here? shouldn't be part of the config file so the user can decide on the fly how to process it?

Reply:
Very good point, adding this into #207 ! For now, it can be specified in the `run.py` directly but nowhere else.